lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -55,6 +55,7 @@ functools32==3.2.3.post2; python_version == '2.7'
 future==0.16.0
 futures==3.2.0; python_version == '2.6' or python_version == '2.7'
 galaxy-sequence-utils==1.1.3
+gxformat2==0.6.0
```

Comment:
Something unexpected is going on here, after merging https://github.com/galaxyproject/galaxy/pull/6807 there should be a merge conflict here, but there isn't ?!

Reply:
This PR had that PRs exact commits - so they don't conflict.  I changed this line twice - once in commits in that PR and again in a commit only in this PR.

Reply:
Alright, I was assuming the context in the github interface would at leastshow a nump from 0.3.1 to 0.6.0 and not show the commits that are already integrated -- although I guess this is a nice "feature" for stale PRs.

Reply:
I'll rebase - I'm sure it will generate a cleaner diff against dev then.